examples/darwin-framework-tool/commands/common/CHIPCommandBridge.mm

```diff
@@ -137,9 +137,13 @@
     ipk = [gNocSigner getIPK];
 
     constexpr const char * identities[] = { kIdentityAlpha, kIdentityBeta, kIdentityGamma };
+    std::string commissionerName = mCommissionerName.HasValue() ? mCommissionerName.Value() : kIdentityAlpha;
     for (size_t i = 0; i < ArraySize(identities); ++i) {
         auto controllerParams = [[MTRDeviceControllerStartupParams alloc] initWithIPK:ipk fabricID:@(i + 1) nocSigner:gNocSigner];
 
+        if (commissionerName.compare(identities[i]) == 0 && mCommissionerNodeId.HasValue()) {
```

Comment:
So commissioner-nodeid will only work in non-interactive mode, basically?  Do we need to do "subscribe with a non-default node id"?

Reply:
Yes, to a certain extent it will not work in interactive mode if the initial params are not provided and will only work on a single controller. For the use case, which is OTA Update, it should be sufficient. At a later time, if we need to set the controller node IDs, we can scope out the work, because we would need to change the "stack startup" portion of the code to start controllers on demand.